src/css/_print.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
+  .video-js > *:not(.vjs-tech):not(.vjs-poster) {
```

Comment:
Haha, that's why I stressed "easy". As part of 5.x, we should still try and support IE8, we don't have to try very hard though :)